gulpfile.js

```diff
@@ -0,0 +1,199 @@
+const gulp = require("gulp");
+const babel = require("gulp-babel");
+const uglify = require("gulp-uglify");
+const rename = require("gulp-rename");
+const sourcemaps = require("gulp-sourcemaps");
+const replace = require("gulp-replace");
+const debug = require("gulp-debug");
+const util = require("gulp-util");
+
+const { rollup } = require("rollup");
+const rollupBabel = require("rollup-plugin-babel");
+const rollupNodeResolve = require("rollup-plugin-node-resolve");
+const rollupCommonjs = require("rollup-plugin-commonjs");
+const rollupJson = require("rollup-plugin-json");
+const rollupReplace = require("rollup-plugin-replace");
+const rollupUglify = require("rollup-plugin-uglify");
+
+const FORMAT_ESM = "esm";
+const FORMAT_CJS = "cjs";
+const FORMAT_UMD = "umd";
+
+const MODULE_NAME = "Eventbrite";
+
+const SOURCE_ENTRY = "src/index.ts";
+
+const FILES_TO_BUILD = [
+  // include all the JavaScript files in src/ directory
+  "src/**/*.@(ts|js)",
+
+  // but exclude the test files
+  "!src/**/__tests__/**"
+];
+
+// When transpiling to ES format, we still use the `env` preset
+// and we want everything transpiled EXCEPT modules
+const ESM_ENV_PRESET = ["@babel/env", { modules: false }];
+
+// When transpiling to UMD, we need the UMD transform plugin.
+// Need to explicitly list the globals unfortunately
+const UMD_TRANSFORM_PLUGIN = [
+  "@babel/plugin-transform-modules-umd",
+  {
+    globals: {
+      index: MODULE_NAME,
+      "isomorphic-fetch": "fetch"
+    },
+    exactGlobals: true
+  }
+];
+
+const _getBabelConfig = format => ({
+  babelrc: false,
```

Comment:
is there too much dev stuff in there that we couldnt reuse it here?

Reply:
Yeah I think there's some sort of bug inheriting it here in gulp. I was trying to override the dev stuff and it just wasn't working, particularly the `modules: false` which we need for ESM